### PR TITLE
Run tests with master and latest napari

### PR DESCRIPTION
**Pull request by goanpeca**
Adds workers to run tests with latest napari, but also with master.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -8,13 +8,14 @@ on:
...
   test:
-    name: ${{ matrix.platform }} ${{ matrix.python }}
+    name: ${{ matrix.platform }} ${{ matrix.python }} (${{ matrix.napari_version }})
     runs-on: ${{ matrix.platform }}
     strategy:
       fail-fast: false
       matrix:
         platform: [ubuntu-latest]
         python: [3.7, 3.8, 3.9]
+        napari_version: ["latest", "master"]
```
Comment (sofroniewn) [inline .github/workflows/test.yml]:
```suggestion
        napari_version: ["stable", "master"]
```
I think we'd used the word stable elsewhere for this (like in URLs), but maybe latest is fine too. Curious if @kne42 has a preference, I don't really mind

**Review by sofroniewn — APPROVED** [review]
LGTM, nice to see build pass against master and latest/ stable

**Comment by alisterburt** [comment]
this looks great, runs the tests on both stable and master of napari as expected.

Are the failures on stable napari expected when running under xvfb and do we know what changed in master to make them run succesfully there? 

I guess a naive solution is to wait for 'stable' to catch up to current 'master' 😆, do you have something in mind @goanpeca ?

**Comment by goanpeca** [comment]
I think 0.4.9 will make now stable pasa

**Comment by melissawm** [comment]
@goanpeca is this still relevant? should this be moved to napari/napari?

**Comment by sofroniewn** [comment]
I don't think we need this anymore - @jni?

**Pull request closed without merging**